changelog/5515.feature.rst

```diff
@@ -0,0 +1,15 @@
+Allow selective auto-indentation of multiline log messages.
+
+Supports command line option ``--log-auto-indent [value]``
+and config option ``log_auto_indent`` to set the default for all logging.
+
+Supports passing kwarg ``extra={"auto_indent": [value]}`` to
+calls to ``log.log()`` to specify auto-indentation behavior for
```

Comment:
What do you mean by `log.log`? Probably you mean logging.log from the stdlib, right? Please try to make this a bit clearer in the changelog entry.

Reply:
You're right, of course. I'll change that